cs/src/Contracts/ClusterDetails.cs

```diff
@@ -0,0 +1,15 @@
+namespace Microsoft.VsSaaS.TunnelService.Contracts;
+
+/// <summary>
+/// Tunnel service cluster details.
+/// </summary>
+public class ClusterDetails {
```

Comment:
I'd suggest an `AzureLocation` property that has the standard Azure location name.